### PR TITLE
User course page

### PR DESCRIPTION
**Pull request by Ernesto905**
Added a user created course page skeleton and linked it to the App component via routing.

### CODE REVIEW

**Review by Rebecca-CYLi — CHANGES_REQUESTED** [review]
Please fill out right side panel to check progress and approvals

**Review by LifeBringer — CHANGES_REQUESTED** [review]
Comments aren't meaningful, also can't see 'development', 'milestone', and 'project' on the right panel.

**Comment by LifeBringer** [comment]
CSS comments are really well structured, good job.

**Comment by LifeBringer** [comment]
Still need to fix permissions on the board and issues. Should be public.

File: src/components/App/App.jsx [hunk 1]
```diff
@@ -1,79 +1,81 @@
-//react imports
-import React from 'react'
-import { BrowserRouter, Routes, Route } from "react-router-dom";
-
-
-
-//styling
```
Comment (LifeBringer) [inline src/components/App/App.jsx:7]: Not a good comment

File: src/components/App/App.jsx [hunk 2]
```diff
@@ -1,79 +1,81 @@
...
-import './App.css'
-
-//Contexts
```
Comment (LifeBringer) [inline src/components/App/App.jsx:10]: Not a good comment

File: src/components/App/App.jsx [hunk 3]
```diff
@@ -1,79 +1,81 @@
...
-import { LearningContextProvider } from '../../contexts/learning';
-import { AuthContextProvider } from '../../contexts/auth'
-import { HomeContextProvider } from '../../contexts/home';
-
-//component imports
```
Comment (LifeBringer) [inline src/components/App/App.jsx:15]: Not a good comment

**Review by LifeBringer — CHANGES_REQUESTED** [review]
Please address the 3 issues, else good to go.

File: src/components/App/App.jsx (hunk 1 above)
Comment (Ernesto905) [reply src/components/App/App.jsx:7]: Are the import comments not descriptive enough, or are they unnecessary for imports?

Comment (LifeBringer) [reply src/components/App/App.jsx:7]: Yup, not descriptive enough and unnecessary.

**Review by LifeBringer — APPROVED** [review]
Changed

**Pull request merged**